bindings/go/mio/mkv.go

```diff
@@ -156,7 +156,7 @@ func (mkv *Mkv) doIdxOp(opcode uint32, key []byte, value []byte,
 
     var rcI C.int32_t
     var op *C.struct_m0_op
-    rc := C.m0_idx_op(mkv.idx, opcode, &k, vPtr, &rcI, flags, &op)
+    rc := C.m0_idx_op(mkv.idx, opcode, &k, vPtr, &rcI, 1, flags, &op)
```

Comment:
I suggest to define a const or an enum to stand for this literal number `1`.
For example:
```
enum {
        DEFAULT_MIN_SUCCESS = 1;
};
```